Tropos/Formatters/TimeOfDay.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+enum TimeOfDay {
+    case Morning
+    case Day
+    case Afternoon
+    case Night
+}
+
+extension NSCalendar {
+    func timeOfDay(forDate date: NSDate) -> TimeOfDay {
+        let components = self.components(.Hour, fromDate: date)
+
+        if components.hour < 4 {
+            return .Night
+        } else if components.hour < 9 {
+            return .Morning
+        } else if components.hour < 14 {
```

Comment:
Isn't everything after noon “afternoon”? Isn't this also missing evening (18:00-00:00)?